### PR TITLE
WIP: ISSUE-47 DistributedServer should not depend on james-server-guice-d…

### PR DESCRIPTION
**Pull request by LanKhuat**
…istributed module.

Continuation of: https://github.com/linagora/openpaas-james/pull/66

Rechecking the code i found out that james-server-guice-distributed module has james-server-cassandra-guice as a dependency so I removed it and added stuffs that were missing.

### CODE REVIEW

File: openpaas-james/apps/distributed/pom.xml [hunk 1]
```diff
@@ -119,10 +123,6 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-guice-data-ldap</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.james</groupId>
-            <artifactId>james-server-guice-distributed</artifactId>
```
Comment (chibenwa) [inline openpaas-james/apps/distributed/pom.xml:124]: No this one is perfectly fine: it is not a product but Guice modules for James as a toolkit...

File: openpaas-james/apps/distributed/pom.xml [hunk 2]
```diff
@@ -108,7 +112,7 @@
             <artifactId>james-server-mailbox-plugin-deleted-messages-vault-guice</artifactId>
         </dependency>
         <dependency>
-            <groupId>${james.groupId}</groupId>
```
Comment (Arsnael) [inline openpaas-james/apps/distributed/pom.xml:111]: keep that :)

File: openpaas-james/apps/distributed/src/main/java/com/linagora/openpaas/james/app/BlobStoreCacheModulesChooser.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
+package com.linagora.openpaas.james.app;
```
Comment (Arsnael) [inline openpaas-james/apps/distributed/src/main/java/com/linagora/openpaas/james/app/BlobStoreCacheModulesChooser.java:1]: Isn't that file copied from apache project? You need to keep the apache license on top then

Idem for similar files

**Comment by chibenwa** [comment]
It is depending on guice-cassandra - that's fine (only modules)

It is not cassandra-guice (product)

File: openpaas-james/pom.xml [hunk 4]
```diff
@@ -196,7 +201,7 @@
                 <version>${james.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.james</groupId>
```
Comment (Arsnael) [inline openpaas-james/pom.xml:199]: keep that

**Comment by LanKhuat** [comment]
We should close this since it's no longer needed?

**Pull request closed without merging**